docs/user/dashboard/lens.asciidoc

```diff
@@ -402,3 +402,19 @@ Pagination in a data table is unsupported. To use pagination in data tables, cre
 ====
 Specifying the color for a single data point, such as a single bar or line, is unsupported.
 ====
+
+[discrete]
+[[is-it-possible-to-inspect-the-elasticsearch-queries-in-Lens]]
+.*Is it possible to inspect the Elasticsearch queries in Lens?*
+[%collapsible]
+====
+Using the Inspector within the editor or in the dashboard it is possible to select the specific request sent by the visualization to Elasticsearch.
+====
+
+[discrete]
+[[is-it-possible-to-isolate-a-single-series-in-a-chart]]
+.*Is it possible to isolate a single series in a chart?*
```

Comment:
Same here, how about 'How to isolate a single series in a chart?' or 'How to view a single chart series in isolation?'

`Press the `shift` key and click on the item in the chart legend to unselect all the other series.`

Again, that's just my perception that it sounds simpler :)